app/models/post.rb

```diff
@@ -1,3 +1,5 @@
 class Post < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  has_many :comments
+  has_many :likes
 end
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the Post model has a method that updates the posts counter for a user as stated in the project requirement.
- [ ] [REQUIRED] Also, let's make sure that the Post model has a method that returns the 5 most recent comments for a given post as stated in the project requirement.